### ISSUE TITLE
Add More Screenshots (and import all of them from USo)

### ISSUE THREAD

**Issue by decembre** [userstyles-world/userstyles.world#17]
I think after you finish the import section, you solve that.
But if you can adding to the edit page a form which allow more screenshots (with a caption field for each), should be great.

**Comment by vednoc** [comment]
@Gusted I wonder, can we do this without JS? For the time being, once can put extra screenshots in notes after Markdown is added, as referenced in #16.

**Comment by Gusted** [comment]
> @Gusted I wonder, can we do this without JS?

> https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file

Why not :D 

Just make sure for some host uploading that we need to configure.

**Comment by vednoc** [comment]
> Why not :D

I meant image carousel. It should be doable, but I'm sure there are tradeoffs.

Also, we should have some kind of limit (max size per image, max images, resolution) as well.

**Comment by Gusted** [comment]
> I meant image carousel. It should be doable, but I'm sure there are tradeoffs.

Why not, just some CSS design for it and server-side very doable.

**Comment by Gusted** [comment]
https://codepen.io/Schepp/pen/WNbQByE :) seems possible.

**Comment by vednoc** [comment]
@Gusted The new USo-archive integration can be extended to also unmarshal the `[]Additional` field. I'll have to find some source with two or more images in order to figure out how to loop over available data.

**Comment by Gusted** [comment]
> I'll have to find some source with two or more images in order to figure out how to loop over available data.

https://33kk.github.io/uso-archive/?search=188453&style=188453

![image](https://user-images.githubusercontent.com/25481501/113339252-82f5fa00-932a-11eb-8195-25179a49eaa0.png)
Pretty straight-forward

**Comment by decembre** [comment]
It seems not all screenshots are imported yet ?

As i can see they are 2 screenshots on USO for:
<a href="https://uso-archive.surge.sh/?author=decembre&amp;sort=updated&amp;style=188453">Youtube WideScreen (New Design Polymer) [USO]</a>

And i see only one On my import:
<a href="https://userstyles.world/style/27">Youtube WideScreen (New Design Polymer) [Userstyles World]</a>
Same for:
<a href="https://userstyles.world/style/28">Userstyles / Greasy Fork Enhancer Dark-Grey [USO]</a>
I don't see all its screenshots (6 images) in the import:
<a href="https://userstyles.world/style/28">Userstyles / Greasy Fork Enhancer Dark-Grey [Userstyles World]</a>

If All screenshots should be imported easily:
Yes, it really need an carousel for them.

Other idea put them or the only one on the left side on a fixed position,
And give them a max-size (smaller than yet) and Maximize if need on hover or click.
You can give a limited height to this Screenshot panel (with overflow: hidden and overfow-y: auto).
Should be interesting:
Like that we can scroll them to verify one thing explained in the Notes.
So divide the screen whit a right place for screenshot(s),
And Title, Description, Notes on the left side.

Others things about screenshots inside the Notes section:
They are too big.
For them too:
Give to them a max-size (smaller than yet) and Maximize if need on hover or click.

**Comment by vednoc** [comment]
> It seems not all screenshots are imported yet ?

Correct. We have yet to implement front- and back-end side of things for importing additional screenshots.

>  Give to them a max-size (smaller than yet) and Maximize if need on hover or click.

Yeah, thought about setting the max-width to 50%. Just like we reduced the size of h1, h2, and h3 headings.
